pkg/csource/csource.go

```diff
@@ -121,11 +122,35 @@ type context struct {
 	calls     map[string]uint64 // CallName -> NR
 }
 
+// This is a kludge, but we keep it here until a better approach is implemented.
```

Comment:
Please also write down what is a better approach, otherwise it's not actionable (it's not obvious to me what should we do instead).